pkg/controller/logstorage/logstorage.go

```diff
@@ -161,6 +162,21 @@ func (r *ReconcileLogStorage) createLogStorage(
 		return reconcile.Result{}, false, finalizerCleanup, err
 	}
 
+	// esInternalCertSecret is a secret handled by ECK
+	if esInternalCertSecret != nil {
+		esPublicSecretComponent := render.NewPassthrough([]client.Object{esInternalCertSecret})
```

Comment:
I suppose it is late now, but wouldn't variadic args be nicer for NewPassthrough?

```
func NewPassthrough(obects ...<object>) Component ()
```